src/libs/EmojiUtils.ts

```diff
@@ -44,6 +44,26 @@ Onyx.connect({
                     return {...emojiWithSkinTones, count: item.count, lastUpdatedAt: item.lastUpdatedAt};
                 })
                 .filter((emoji): emoji is FrequentlyUsedEmoji => !!emoji) ?? [];
+
+        // On AddComment API response, each variant of the same emoji (with different skin tones) is
+        // treated as a separate entry due to unique emoji codes for each variant.
+        // So merge duplicate emojis, sum their counts, and use the latest lastUpdatedAt timestamp, then sort accordingly.
+        const emojiMap = new Map<string, FrequentlyUsedEmoji>();
```

Comment:
It's obvious that this a Map, let's call this `frequentlyUsedEmojiCodesToObjects`